### ISSUE TITLE
Cannot push package "Unable to write data to the transport connection: An existing connection was for cibly closed by the remote host"

### ISSUE THREAD

**Issue by Greenstone3672** [NuGet/Home#815]
When trying to push a package to my company's NuGet server, I receive the error in the title.  The only information I was able to scrounge up on this was from another issue here: https://github.com/NuGet/NuGetGallery/issues/391

However, unlike the poster from that thread my package file isn't as large (1.5 MB), so I'm not sure if it's a similar issue.  I'm using version 2.8.3 of nuget.core and nuget.server.  The command I'm using looks like:

nuget push [package name] [api key] [source]

**Comment by yishaigalatzer** [comment]
Could you try something for us - Please use nuget 2.7 and tell us if this works for you

**Comment by Greenstone3672** [comment]
I tried 2.7 and it still gave me the same issue.  I also tried a smaller test package (261 KB) and got the same error.

This server is infrastructure that I inherited from a coworker who left the company, so I could also be missing something obvious.  I'm going to learn more about the NuGet server itself and Azure (where it's hosted) to hopefully not waste your time, although I would certainly take any advice you happen to have.
